### PR TITLE
2023 00029 xortigate draft case file

### PR DESCRIPTION
**Pull request by 82BBB00**
Attempt to add new draft case file for feedback. DIVD-2023-00029

### CODE REVIEW

File: _cases/2023/DIVD-2023-00029.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  end:
+  event: "Fox-IT/NCC Group shares data of more vulnerable devices with DIVD"
+- start: 2023-07-17
+  end:
+  event: "First version of this casefile"
+---
+
+## Summary
+Following previous incident FG-IR-22-398 / CVE-2022-42475 published on January 11, 2023 (known by us as DIVD-2022-00063) where a heap-based buffer overflow in FortiOS SSL VPN with exploitation was observed in the wild, the Fortinet Product Security Incident Response Team (PSIRT) proactively initiated a code audit of the SSL-VPN module This audit, together with a responsible disclosure from a third-party researcher, led to the identification of this new critical SSL-VPN RCE Vulnerability.
+According to a blog posted by Bishop Fox there were on June 30, 2023 nearly 490,000 affected SSL VPN interfaces exposed on the internet and roughly 69% of them were around that time unpatched.
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00029.md:74]: According to a blog, posted by BishopFox, there were nearly 490,000 affected SSL VPN interfaces exposed on the internet on June 30th of 2023. Roughly 69% of them were at that time unpatched.

File: _cases/2023/DIVD-2023-00029.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+## What you can do
+Upgrade your SSL VPN instance to the newest version or make us of the work-around: disable SSL VPN or only allow whitelisted IPs.
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00029.md:77]: "newest version" -> latest version
"make us" -> make use

File: _cases/2023/DIVD-2023-00029.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## What we are doing
+DIVD is currently scanning for vulnerable instances connected to the public Internet. Furthermore, we received new data from Fox-it /NCC Group that we will use. Owners of vulnerable systems will receive a notification with instructions to update their system.
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00029.md:80]: Internet -> random capital I
/NCC -> add space between / and NCC

File: _cases/2023/DIVD-2023-00029.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+
+{% include timeline.html %}
+
+## More information
+* {% cve CVE-2023-24489 %}
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00029.md:85]: again wrong CVE number?

File: _cases/2023/DIVD-2023-00029.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+* [Fortinet Security Advisory](https://www.fortiguard.com/psirt/FG-IR-23-097)
+* [Fortinet PSIRT Blog CVE-2023-24489](https://www.fortinet.com/blog/psirt-blogs/analysis-of-cve-2023-27997-and-clarifications-on-volt-typhoon-campaign)
```
Comment (Lennaert89) [inline _cases/2023/DIVD-2023-00029.md:87]: wrong cve number?

**Review by Lennaert89 — CHANGES_REQUESTED** [review]

File: _cases/2023/DIVD-2023-00029.md (hunk 3 above)
Comment (Ralphhorn) [reply _cases/2023/DIVD-2023-00029.md:80]: fixed and changed text

**Review by Lennaert89 — APPROVED** [review]

**Pull request merged**